pkg/models/clusterConfig.go

```diff
@@ -3,6 +3,7 @@
 package models
 
 type ClusterConfig struct {
-	KeyPair  map[string]string   // maps key pair name to cert path
-	Clusters map[string][]string // maps clusterName to nodeID list
+	KeyPair   map[string]string   // maps key pair name to cert path
+	Clusters  map[string][]string // maps clusterName to nodeID list
+	GCPConfig map[string]string   // GCP Only: project name and filepath to service account JSON key
```

Comment:
Ok I understand a bit better. The map keys are constants from contants package. Most probably we want to have two fields for that here, instead of a map? Probably a struct for GCPConfig, with the two fields, and use the struct instead of the map